apps/sveltekit-example-app/src/routes/account/sign-up/+page.svelte

```diff
@@ -20,18 +20,18 @@
 
     <form class="space-y-6" method="POST" novalidate use:enhance>
       <div>
-        <label class="block" for="email">Email</label>
+        <label class="block" for="username">Email</label>
         <div class="mt-2">
           <input
-            aria-invalid={$errors.email ? 'true' : undefined}
-            bind:value={$formData.email}
-            data-invalid={$errors.email ? 'true' : undefined}
-            name="email"
+            aria-invalid={$errors.username ? 'true' : undefined}
```

Comment:
switched this over to username as this name is more correct.  Currently require username to be a email address.